### DISCUSSION TITLE
Websocket client support

### DISCUSSION THREAD

**Discussion by rosssaunders** [omnigres/omnigres#344] [Ideas]
Hi,

Do you have any plans or views on adding support for websocket clients so that from Postgres you can start listening to external websocket streams and invoke a callback into a Postgres function to handle the request?

It seems like you could achieve this using a background worker process and a postgres extension.

Thanks,

Ross

**Comment by yrashk** [comment]
Hi Ross,

Yes, it is in the short-term plan.
